backend/src/main/java/gov/cdc/usds/simplereport/config/DevSecurityConfiguration.java

```diff
@@ -19,7 +21,7 @@
 @Profile(BeanProfiles.NO_SECURITY)
 @Configuration
 @ConditionalOnWebApplication
-public class DevSecurityConfiguration extends WebSecurityConfigurerAdapter {
+public class DevSecurityConfiguration extends WebSecurityConfigurerAdapter implements WebMvcConfigurer {
```

Comment:
We already have one WebMvcConfigurer in the config—will they both be picked up? Only an issue in dev and demo, I guess, but considering what's defined in that config, it would be a quite significant issue!

Reply:
Yes – when more than one WebMvcConfigurer is present, Spring just [loops over them](https://github.com/spring-projects/spring-framework/blob/3a0f309e2c9fdbbf7fb2d348be861528177f8555/spring-webmvc/src/main/java/org/springframework/web/servlet/config/annotation/WebMvcConfigurerComposite.java#L86)